### PR TITLE
修改部分难以理解的语法、冗长的语法、警告信息

### PR DESCRIPTION
**Pull request by jimcat8**
## Author checklist (to be completed by original Author)
- [x] Is this document a good fit for the Rocky project ?
- [x] Is this a non-English contribution? 
- [ ] Title and Author MetaTags have been inserted into the document 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [ ] 1st Pass (Check that document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Basic Editorial Review)
- [ ] 4th Pass (Detailed Editorial Review and Peer Review)
- [ ] 5th Pass (Include document in TOC)
- [ ] Final pass/approval (Final Review)

### CODE REVIEW

File: docs/guides/installation.zh_CN.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: 安装 Rocky Linux
 
 本文介绍在独立系统上安装 64 位 rocky Linux 发行版的详细步骤。
```
Comment (justasojourner) [inline docs/guides/installation.zh_CN.md:7]: Should be 'Rocky Linux'

File: docs/guides/installation.zh_CN.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ title: 安装 Rocky Linux
...
 
-本文将使用从 rocky 项目网站下载的操作系统安装程序映像执行服务器类安装。以下部分将介绍安装和自定义步骤。
+本文将使用从 rocky 官方网站下载的iso镜像，执行最小化安装。以下部分将介绍安装和自定义步骤。
```
Comment (justasojourner) [inline docs/guides/installation.zh_CN.md:9]: Should be full name in initial  uppercase - 'Rocky Linux'

File: docs/guides/installation.zh_CN.md (hunk 1 above)
Comment (justasojourner) [inline docs/guides/installation.zh_CN.md:7]: Should be 'Rocky Linux'

**Comment by justasojourner** [comment]
Hi @jimcat8 

Thanks for your recent update. I had a look at it, just a few points to share:

1. I see you created a new file `README_cn.md` the convention (technical requirement actually) for file naming is 
FILENAME.XX.md where 'FILENAME' is the base filename, 'XX' is the language code which **must** be **exactly (including case sensitive)** one of the language codes supported by the Rocky docs website. From the configuration file for the website software, MkDocs:
```
      languages:
          en: English
          de: Deutsch
          fr: Français
          es: Español
          jp: 日本語
          zh_CN: 简体中文
          sv: Svenska
```
Just to share the technical reason for this... when MkDocs does a `build` of the website it goes through all documents and parses the file name (by working through the '.'). It gets the base filename, then if there is a language code it creates a file for that language and places the file into the specific language directory tree of the docs directory tree.

```
.
├── en
│   ├── books
│   ├── guides
│   ├── labs
│   └── release_notes
├── fr
│   ├── books
│   ├── guides
│   ├── labs
│   └── release_notes
└── zh_CN
    ├── books
    ├── guides
    ├── labs
    └── release_notes
```

If there is not a translated version of a file, for example `filename.zh_CN.md`, then it will fall back to the default language (en) version and copy that into the specific language tree. Each language tree then has a complete copy of the website. Localised versions (for example zh_CN) will have a mixture of translated files and English files (for documents that have not been translated).

So `README_cn.md` should be `README.zh_CN.md`.

2. However need to advise that the Contributions home page is being considered for a revamp and the page will very likely change _so at this time it is best not to put any work into it_.

3. There is no need to put language navigation links in files
```
**[简体中文](README_cn.md)**
**[English](README.md)**

# Contribution Guide
## Introduction
``` 
Because the MkDocs system puts different language files into the appropriate language tree and from the common menu will load a localised file - if one exists.

For example if I choose the English language version of the website I will land in 
https://docs.rockylinux.org/en/
if you choose Simplified Chinese you will land in
https://docs.rockylinux.org/zh_CN/

The top level menus will be the same but if choose the _same_ menu option we will automatically land in _different_ directory trees (different documents) depending on which language we are viewing. So there is no need to put manual navigators at the top of each document.

So, long comment, sorry — basically though re: the latest changes you made it is highly possible that the new document will not be folded in because the base contributions page will likely be changing.

**Comment by justasojourner** [comment]
Hi @jimcat8 

Found a couple of problems with the home page, basically they come down to the point that MkDocs requires Markdown directives to be in English.

## Menu title

Displays as this:
Screen Shot 2021-09-26 at 11 09 06

This is because the meta tag is this
```
---
标题：首页
---
```

But should be this
```
---
title：Home
---
```

Or, if translation of the top level menu text is desired, this:
```
---
title：首页
---
```

But the meta tag `title` must be in English for MkDocs to understand it.

## Admonition

The admonition in the page currently looks like this:
Screen Shot 2021-09-26 at 10 55 38

but should be this:
Screen Shot 2021-09-26 at 10 55 23

The tag for admonitions must not be translated, and the `!!!` must not have any spaces and the admonition text must be indented by four spaces

So not this:
```
！！！笔记
    并非所有功能都适用于所有语言，这是可用的底层 MkDocs 模块的限制。
```

but this :
```
!!! note
    并非所有功能都适用于所有语言，这是可用的底层 MkDocs 模块的限制。
```

Cross ref. the original:
```
!!! note
    Not all functionality may be available in all languages, this is a limitation of the underlying MkDocs modules available.
```

**Comment by sspencerwire** [comment]
Thanks, @jimcat8 for the time and effort you have put into this. I wanted to let you know that I'm going to merge the Installation Guide and the Index changes. We are going to be making some changes to the Contribution section that will involve the README.md files. Because of the language selector at the top of the documentation website, you should not need to link to the two version within each README, but rather than changing that, based on the changes that are coming, I'm going to delete those two files in the PR for now.  Keep watch on the site for upcoming changes and I will welcome your edits to those pages when the time comes. Thanks again for your work! Keep it up! We need more translators!

**Pull request merged**